libminifi/src/core/ProcessSession.cpp

```diff
@@ -868,6 +868,8 @@ void ProcessSession::commit() {
     _clonedFlowFiles.clear();
     _deletedFlowFiles.clear();
     _originalFlowFiles.clear();
+
+    _transferRelationship.clear();
```

Comment:
When we commit the FlowFiles they are no longer needed, but they take up a lot of memory.

Reply:
Nice catch!